### DISCUSSION TITLE
Where should we put entities in the DDD architecture?

### DISCUSSION THREAD

**Discussion by Merwan-J** [Merwan-J/AskAnything#58] [Q&A] [answered]
I have been researching a lot and I have seen different forms of DDD architecture. looks like people tweak the architecture to their need.

**Comment by amanyih** [comment]
I think it doesn't matter. We can put it either on infrastructure or domain layer

**Comment by yeabtsega07** [comment, marked as answer]
As far as I know. We should put it in the domain layer under a folder called local
